site/docs/hosting/cloudflare-workers-nodejs.md

```diff
@@ -58,35 +198,129 @@ npm install grammy
 
 Edit `src/index.js` or `src/index.ts`, and write this code inside:
 
-```ts
-import { Bot, webhookCallback } from "grammy";
+```ts{11,28-29,38,40-42,44}
+/**
+ * Welcome to Cloudflare Workers! This is your first worker.
+ *
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your worker in action
+ * - Run `npm run deploy` to publish your worker
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+
+import { Bot, Context, webhookCallback } from "grammy";
+
+export interface Env {
+  // Example binding to KV. Learn more at https://developers.cloudflare.com/workers/runtime-apis/kv/
+  // MY_KV_NAMESPACE: KVNamespace;
+  //
+  // Example binding to Durable Object. Learn more at https://developers.cloudflare.com/workers/runtime-apis/durable-objects/
+  // MY_DURABLE_OBJECT: DurableObjectNamespace;
+  //
+  // Example binding to R2. Learn more at https://developers.cloudflare.com/workers/runtime-apis/r2/
+  // MY_BUCKET: R2Bucket;
+  //
+  // Example binding to a Service. Learn more at https://developers.cloudflare.com/workers/runtime-apis/service-bindings/
+  // MY_SERVICE: Fetcher;
+  //
+  // Example binding to a Queue. Learn more at https://developers.cloudflare.com/queues/javascript-apis/
+  // MY_QUEUE: Queue;
+  BOT_INFO: string;
+  BOT_TOKEN: string;
+}
 
-// The following line of code assumes that you have configured the secrets BOT_TOKEN and BOT_INFO.
-// See https://developers.cloudflare.com/workers/platform/environment-variables/#secrets-on-deployed-workers.
-// The BOT_INFO is obtained from `bot.api.getMe()`.
-const bot = new Bot(BOT_TOKEN, { botInfo: BOT_INFO });
+export default {
+  async fetch(
+    request: Request,
+    env: Env,
+    ctx: ExecutionContext,
+  ): Promise<Response> {
+    const bot = new Bot(env.BOT_TOKEN, { botInfo: JSON.parse(env.BOT_INFO) });
+
+    bot.command("start", async (ctx: Context) => {
+      await ctx.reply("Hello World!");
```

Comment:
```suggestion
      await ctx.reply("Hello, world!");
```

Reply:
Could you please sync these changes to all languages?

Reply:
I didn't realize there were other files.

Ok. No problem.

Reply:
> but in the UK file, "Hello World" is translated.
It's okay to leave them, then.